rules/cyberark/privilege_escalation_cyberarkpas_error_audit_event_promotion.toml

```
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/06/23"
+maturity = "production"
+updated_date = "2021/06/23"
+min_stack_version = '7.14.0'
```

this is exciting

---

getting ahead of the lint change
```suggestion
min_stack_version = "7.14.0"
```